src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  switch (true) {
+    case days < 3:
+      total = 40 * days;
+
+      return total;
+    case days >= 3 && days < 7:
```

Comment:
```suggestion
    case days < 7:
```
Якщо перша перевірка пройшла, то до цієї вже функція не дійде. А якщо перша перевірка не пройшла, то значить днів точно не менше трьох. Тому одну з умов можна прибрати